Felienne/hedy
# [BUG FIX] Correctly set keyword language on custom adventure page

**Description**
In this PR we fix an issue where the custom adventure page was not set with the correctly keyword language. This was due to the `window.State.keyword_language` not being set on page call. We fix this by overwriting the script block. 

Note that is not a really nice solution with copying and overwriting the whole script block. It works, for now, but we should clean-up the script blocks on the whole website (there is already an issue for that).

**Fixes**
This PR fixes #2504.

**How to test**
Make sure you are a teacher and have your keyword language set to a non-english language. Create a custom adventure and use some localized keywords. Verify that the syntax highlighting works as expected. Both within the preview window as well as on the dedicated `view adventure` page.
